change_color.sh

```diff
@@ -229,7 +229,7 @@ done
       #-e 's/%INACTIVE_TXT_FG%/'"$INACTIVE_TXT_FG"'/g' \
       #-e 's/%INACTIVE_MENU_FG%/'"$INACTIVE_MENU_FG"'/g' \
 
-sed -i -e 's/^$material_radius: .px/$material_radius: '"$ROUNDNESS"'px/g' ./src/_sass/gtk/_variables.scss
+sed -i -e 's/^$material_radius: .px/$material_radius: '"$ROUNDNESS"'px/g' ./src/_sass/_variables.scss
```

Comment:
```
for FILEPATH in ./src/_sass/*/_variables.scss ; do
    sed -i -e 's/^$material_radius: .px/$material_radius: '"$ROUNDNESS"'px/g' "${FILEPATH}"
done
```

Reply:
@actionless , `$material_radius` is not specified in `./src/_sass/gnome-shell/_variables.scss`

Reply:
@actionless Don't worry, I won't add `$material_radius` into `./src/_sass/gnome-shell/_variables.scss` :smile: